app/views/shared/_sign_in_form.html.erb

```diff
@@ -3,19 +3,13 @@
     <%= f.label(:email, _('Email'), class: 'control-label') %>
     <%= f.email_field(:email, class: 'form-control', "aria-required": true) %>
   </div>
-  <div class="checkbox">
-    <%= label_tag 'remember_email', raw("#{check_box_tag 'remember_email'}#{_('Remember email')}") %>
-  </div>
   <div class="form-group">
     <%= f.label(:password, _('Password'), class: 'control-label') %>
     <%= f.password_field(:password, class: 'form-control', "aria-required": true) %>
   </div>
   <div class="checkbox">
-    <label for="passwords_toggle">
-      <input type="checkbox" id="passwords_toggle" /><%= _('Show password') %>
-    </label>
+    <%= label_tag 'remember_email', raw("#{check_box_tag 'remember_email'}#{_('Remember email')}") %>

```

Comment:
since show password is not longer applicable here. We should also clean lib/assets/javascripts/views/shared/sign_in_form.js (L3 and L7)